user/views.py

```diff
@@ -132,18 +131,19 @@ def post(self, request):
            
 
 class ResetpasswordView(APIView):
-    permission_classes = [permissions.IsAuthenticated]
+    authentication_classes = [authentication.TokenAuthentication]
+    serializer_class = ResetpasswordSerializer
     
     def post(self, request):
+        email = request.data.get('email')
         password = request.data.get('password')
-        new_password = request.data.get('new_password')
-        confirm_password = request.data.get('confirm_password')
-        user = User.objects.filter(email=email)
-        if user.check_password(email):
-            password.validate_password(password=request.data, user=User)
-            return password
-        
-        return Response(({'details', 'create a password'}))
-        
-        
+        user = User.objects.get(email=email)
```

Comment:
Why we are not using try except or get_object_or_404 here ?

Reply:
fixed